packages/run-protocol/src/vaultFactory/vaultKit.js

```diff
@@ -7,14 +8,14 @@ import { makeVaultHolder } from './vaultHolder.js';
  * Create a kit of utilities for use of the vault.
  *
  * @param {Vault} vault
- * @param {Promise<Notifier<import('./vaultManager').AssetState>>} assetSubscriber
+ * @param {Subscriber<import('./vaultManager').AssetState>} assetSubscriber
  */
 export const makeVaultKit = (vault, assetSubscriber) => {
   const { holder, helper } = makeVaultHolder(vault);
   const vaultKit = harden({
     publicNotifiers: {
       vault: holder.getNotifier(),
-      asset: assetSubscriber,
+      asset: makeNotifierFromSubscriber(assetSubscriber),
```

Comment:
we make this notifier just once? not a new prefix-lossy one on demand?

Reply:
right, this is interim. the contract shouldn't be making Notifiers at all because that costs state tracking. Change in next PR.